lib/qa/authorities/assign_fast_subauthority.rb

```diff
@@ -0,0 +1,31 @@
+# Encapsulate information about assignFAST subauthorities
+module Qa::Authorities::AssignFastSubauthority
+
+  # Hash subauthority name used in route => 'index' used by API
+  Subauthorities = {
+    'all'        => 'suggestall',
+    'personal'   => 'suggest00',
+    'corporate'  => 'suggest10',
+    'event'      => 'suggest11',
+    'uniform'    => 'suggest30',
+    'topical'    => 'suggest50',
+    'geographic' => 'suggest51',
+    'form_genre' => 'suggest55',
+  }
+
+  # Get a list of subauthorities by name
+  #
+  # @return [Array<String>]
+  def get_authorities_keys
+    return Subauthorities.keys
+  end
+
+  # Get an API index name from an English name
+  #
+  # @param [String] English name
+  # @return [String] index name
+  def get_index_for_authority authority
```

Comment:
Idiomatic ruby method names don't begin with `get`, so how about calling this `index_for_authority`?